benchmarks/Dapper.Tests.Performance/Benchmarks.Norm.cs

```diff
@@ -0,0 +1,42 @@
+﻿#if NET4X 

```

Comment:
Why not "#if !NET4X"? then you wouldn't need the empty "then" branch nor "else" statement.

Also, the package reference comes under condition "'$(TargetFramework)' == 'netcoreapp3.1'" which means that if TargetFramework would be set to 'net5.0' then the reference will be missing, but the NormBenchmarks class would be presented and hence there will be a compilation error. So I would propose to have the benchmark file itself in .csproj to be under the same condition that is used for PackageReference instead of having the "#if" directive inside the file.